src/app/installer/create-wallet/passphrase/passphrase.service.ts

```diff
@@ -39,10 +39,11 @@ export class PassphraseService {
     return this.validWords.indexOf(word) !== -1;
   }
 
-  importMnemonic(words: string[], password: string) {
-    const params = [words.join(' '), password];
+  importMnemonic(words: string[], password: string, isLegacy: boolean = false) {
+    const params = [words.join(' '), password, false , 'Master Key', 'Default Account', '0', isLegacy];
     if (!password) {
-      params.pop();
+      // If no password we use default value
+      params[1] = '';
```

Comment:
Why was this changed?

Reply:
It was using params.pop() before, now as we are passing all the parameters, if we using pop, we basically was erasing the isLegacy param.

Reply:
Ok